torch/csrc/distributed/c10d/ProcessGroupGloo.cpp

```diff
@@ -929,7 +929,7 @@ class AsyncBroadcastCUDAWork : public AsyncBroadcastWork {
 
 } // namespace
 
-c10::intrusive_ptr<ProcessGroup::Work> ProcessGroupGloo::broadcast(
+c10::intrusive_ptr<ProcessGroup::Work> ProcessGroupGloo::broadcast_impl(
```

Comment:
Our solution can't rely on changing all subclasses since there are third-party implementations of the ProcessGroup interface that we don't control.